tests/cypress/support/commands.js

```diff
@@ -1778,3 +1826,11 @@ Cypress.Commands.add('clickSaveAnnotationView', () => {
     cy.get('button').contains('Save').click();
     cy.get('button').contains('Save').trigger('mouseout');
 });
+
+Cypress.Commands.add('clickUndo', () => {
+    cy.contains('.cvat-annotation-header-button', 'Undo').click();
+});
+
+Cypress.Commands.add('clickFirstFrame', () => {
+    cy.get('.cvat-player-first-button').click();
+});
```

Comment:
I do not think one-line commands make sense. 

It may be considered as encapsulation approach, what is basically might be a good idea, however if you add it here, we must replace all corresponding occurrences and implement the same for other buttons, what is definitely not related to this pull request.